english/buy-gameboy/buy-gameboy.md

```diff
@@ -1,3 +1,5 @@
 you realize that Pokemon is great and can still help you with your troubles, 
 you find a man at a flea market selling Pokemon Red,
-Blue, and Yellow all for 20 dollars along with a Gameboy. You buy it. 
\ No newline at end of file
+Blue, and Yellow all for 20 dollars along with a Gameboy. You buy it. 
+
+[But it's not a game, boy](./english/ninja/ninja.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.